src/components/tag/_tag.scss

```diff
@@ -20,5 +19,11 @@
 
     text-decoration: none;
     text-transform: uppercase;
+
+    &--inactive {
+      background-color: $govuk-grey-2;
```

Comment:
Do we have the concept of a tag being inactive? Are tags used to reflect an active/inactive status?
Using this grey shade as a background colour for a white text results in low contrast ratio (fails AA level). If we decide to use a grey maybe we should try `$govuk-grey-1` (which passes AAA level)?

Reply:
Good questions – one for @timpaul perhaps?

Another place where we're using the nested / shorthand thing which makes it hard to search for things. (Should we add a standard or linting rule for this?)